test/functional/test_framework/test_framework.py

```diff
@@ -656,7 +659,7 @@ def _initialize_chain_clean(self):
     def skip_if_no_py3_zmq(self):
         """Attempt to import the zmq package and skip the test if the import fails."""
         try:
-            import zmq  # noqa
+            import zmq  # type: ignore  # noqa
```

Comment:
The `type: ignore` here should no longer be required?

Reply:
You are right. Fixed.